### PR TITLE
docs(visible): Reinforce document

### PR DESCRIPTION
**Pull request by happyhj**
## Issue
#296 

## Details
Improve descriptions on jsdoc

## Preferred reviewers
@naver/egjs-dev

### CODE REVIEW

File: src/visible.js [hunk 1]
```diff
@@ -64,14 +64,14 @@ eg.module("visible", ["jQuery", eg, document], function($, ns, doc) {
...
-		 * @ko target들을 갱신한다.
+		 * Update internal target list
+		 * @ko 내부 target 리스트를 갱신한다.
 		 * @method eg.Visible#refresh
 		 * @return {eg.Visible} instance of itself<ko>자신의 인스턴스</ko>
 		 *
 		 * @remark
-		 * If targets was added or removed, must be called to refresh
-		 * <ko> 확인 대상이 영역 안에 추가 된 경우, 또는 확인 대상이 영역 안에서 삭제 된 경우, 영역 내의 확인 대상을 이 메소드를 호출하여 갱신해야한다. <ko>
+		 * If targets was added or removed from dom tree, must call refresh method to update internal target list.
```
Comment (netil) [inline src/visible.js]: It's not a big deal, but abbreviated word would be better write in capital.

File: src/visible.js [hunk 2]
```diff
@@ -174,8 +175,8 @@ eg.module("visible", ["jQuery", eg, document], function($, ns, doc) {
 				}
 			}
 			/**
-			 * Trigger when the target elements are visible or hidden based on the base area.
-			 * @ko 기준 영역을 기준으로 보이는 엘리먼트와 사라진 엘리먼트가 변경된 경우 발생하는 이벤트
+			 * Triggered then the visibility status is changed compared with the status from when last check metod called.
+			 * @ko check 를 통해 마지막으로 확인한 결과와 비교하여 보이는 엘리먼트와 사라진 엘리먼트가 변경된 경우 발생하는 이벤트
```
Comment (netil) [inline src/visible.js]: 검사 sounds better instead 'check'.

File: src/visible.js [hunk 3]
```diff
@@ -64,14 +64,14 @@ eg.module("visible", ["jQuery", eg, document], function($, ns, doc) {
 			this.refresh();
 		},
 		/**
-		 * Update targets
-		 * @ko target들을 갱신한다.
+		 * Update internal target list
+		 * @ko 내부 target 리스트를 갱신한다.
```
Comment (jongmoon) [inline src/visible.js]: @happyhj 
내부 target 리스트 ==> Visibility 대상 목록

I don't have translation idea for a word `visibility`. How about you?

File: src/visible.js [hunk 4]
```diff
@@ -91,8 +91,9 @@ eg.module("visible", ["jQuery", eg, document], function($, ns, doc) {
 			return this.refresh();
 		},
 		/**
-		 * Checks if the target elements has been changed.
-		 * @ko target들이 변경했는지 확인한다.
+		 * Checks if the target elements' visibility has been changed.
+		 * "change" event will be triggered if there is a change.
+		 * @ko target들의 visibility 가 변경되었는지 확인한다. 변경이 존재할 경우 "change" 이벤트가 발생한다.
```
Comment (jongmoon) [inline src/visible.js]: @happyhj 
target 들의 ==> 대상 목록의

File: src/visible.js (hunk 2 above)
Comment (jongmoon) [inline src/visible.js]: @happyhj 
It's ok to remove `check 를 통해`.

File: src/visible.js [hunk 5]
```diff
@@ -65,12 +65,12 @@ eg.module("visible", ["jQuery", eg, document], function($, ns, doc) {
 		},
 		/**
 		 * Update internal target list
-		 * @ko 내부 target 리스트를 갱신한다.
+		 * @ko Visibility 검사 대상 엘리먼트 목록을 갱신한다.
```
Comment (jongmoon) [inline src/visible.js]: @happyhj 
**Suggest**
Visibility 를 검사할 대상 엘리먼트 목록

File: src/visible.js [hunk 6]
```diff
@@ -174,8 +175,8 @@ eg.module("visible", ["jQuery", eg, document], function($, ns, doc) {
 				}
 			}
 			/**
-			 * Trigger when the target elements are visible or hidden based on the base area.
-			 * @ko 기준 영역을 기준으로 보이는 엘리먼트와 사라진 엘리먼트가 변경된 경우 발생하는 이벤트
+			 * Triggered then the visibility status is changed compared with the status from when last check metod called.
```
Comment (netil) [inline src/visible.js]: Quite difficult to understand. How about this?
> Event is triggered when visible and hidden elements changes are detected, compared with last visibility status.

Comment (happyhj) [inline src/visible.js]: It looks better. I'll apply it. thanks

**Comment by netil** [comment]
LGTM

**Comment by jongmoon** [comment]
LGTM

**Pull request merged**